### ISSUE TITLE
[BUG]: Aspect Model Editor v5 leads to a white screen

### ISSUE THREAD

**Issue by tom-rm-meyer-ISST** [eclipse-esmf/esmf-aspect-model-editor#131]
**How: List of performed steps**
- Download latest release (v5.1.1-linux) with gclib version 2.31
- Extract the zip
- Start the application via "aspect-model-editor" 
- Migration of workspace popup appears > when trying to click something, the windows gets white and can't be closed via 'x' control

![image](https://github.com/eclipse-esmf/esmf-aspect-model-editor/assets/116013795/8f5b7d38-dcdb-4772-b787-9fd42d0e6012)

**What: Actual result vs expected result**
The windows should be usable

**Where: (e.g. component, version, url, your system info)**
- linux distribution

**Input (files/screenshots)**

**Output (files/screenshots)**

**Additional context (e.g. references)**
- Also moved the workspace to not run into the migration popup. Then normal view lights up for ~ a second before switching to the white screen
- also tried version 5.0.0
- Updated Ubuntu last time ([see issue](https://github.com/eclipse-esmf/esmf-aspect-model-editor/issues/56)) to 22.04.3 LTS with gclib 2.35

**Comment by tom-rm-meyer-ISST** [comment]
Hi @michelu89, thanks for checking. Outputs as wished.

Sidenote: As you see I'm running the system in VM Ware Workstation virutalization software.

``` shell
lshw -short (not sudo)

H/W path           Device      Class          Description
=========================================================
                               system         Computer
/0                             bus            Motherboard
/0/0                           memory         16GiB System memory
/0/1                           processor      AMD Ryzen 7 PRO 5850U with Radeon Graphics
/0/2                           processor      AMD Ryzen 7 PRO 5850U with Radeon Graphics
/0/3                           processor      AMD Ryzen 7 PRO 5850U with Radeon Graphics
/0/4                           processor      AMD Ryzen 7 PRO 5850U with Radeon Graphics
/0/5                           processor      AMD Ryzen 7 PRO 5850U with Radeon Graphics
/0/6                           processor      AMD Ryzen 7 PRO 5850U with Radeon Graphics
/0/7                           processor      AMD Ryzen 7 PRO 5850U with Radeon Graphics
/0/8                           processor      AMD Ryzen 7 PRO 5850U with Radeon Graphics
/0/100                         bridge         440BX/ZX/DX - 82443BX/ZX/DX Host bridge
/0/100/1                       bridge         440BX/ZX/DX - 82443BX/ZX/DX AGP bridge
/0/100/7                       bridge         82371AB/EB/MB PIIX4 ISA
/0/100/7/0                     system         PnP device PNP0c02
/0/100/7/1                     system         PnP device PNP0b00
/0/100/7/2                     input          PnP device PNP0303
/0/100/7/3                     generic        PnP device VMW0003
/0/100/7/4                     system         PnP device PNP0103
/0/100/7/5                     communication  PnP device PNP0501
/0/100/7/6                     system         PnP device PNP0c02
/0/100/7.1                     storage        82371AB/EB/MB PIIX4 IDE
/0/100/7.3                     bridge         82371AB/EB/MB PIIX4 ACPI
/0/100/7.7                     generic        Virtual Machine Communication Interface
/0/100/f           /dev/fb0    display        SVGA II Adapter
/0/100/10                      storage        53c1030 PCI-X Fusion-MPT Dual Ultra320 SCSI
/0/100/11                      bridge         PCI bridge
/0/100/11/0                    bus            USB1.1 UHCI Controller
/0/100/11/1        ens33       network        82545EM Gigabit Ethernet Controller (Copper)
/0/100/11/2        card0       multimedia     ES1371/ES1373 / Creative Labs CT2518
/0/100/11/3                    bus            USB2 EHCI Controller
/0/100/11/4        scsi3       storage        SATA AHCI controller
/0/100/11/4/0.0.0  /dev/cdrom  disk           VMware SATA CD01
/0/100/15                      bridge         PCI Express Root Port
/0/100/15/0                    bus            USB3 xHCI 1.0 Controller
/0/100/15.1                    bridge         PCI Express Root Port
/0/100/15.2                    bridge         PCI Express Root Port
/0/100/15.3                    bridge         PCI Express Root Port
/0/100/15.4                    bridge         PCI Express Root Port
/0/100/15.5                    bridge         PCI Express Root Port
/0/100/15.6                    bridge         PCI Express Root Port
/0/100/15.7                    bridge         PCI Express Root Port
/0/100/16                      bridge         PCI Express Root Port
/0/100/16.1                    bridge         PCI Express Root Port
/0/100/16.2                    bridge         PCI Express Root Port
/0/100/16.3                    bridge         PCI Express Root Port
/0/100/16.4                    bridge         PCI Express Root Port
/0/100/16.5                    bridge         PCI Express Root Port
/0/100/16.6                    bridge         PCI Express Root Port
/0/100/16.7                    bridge         PCI Express Root Port
/0/100/17                      bridge         PCI Express Root Port
/0/100/17.1                    bridge         PCI Express Root Port
/0/100/17.2                    bridge         PCI Express Root Port
/0/100/17.3                    bridge         PCI Express Root Port
/0/100/17.4                    bridge         PCI Express Root Port
/0/100/17.5                    bridge         PCI Express Root Port
/0/100/17.6                    bridge         PCI Express Root Port
/0/100/17.7                    bridge         PCI Express Root Port
/0/100/18                      bridge         PCI Express Root Port
/0/100/18.1                    bridge         PCI Express Root Port
/0/100/18.2                    bridge         PCI Express Root Port
/0/100/18.3                    bridge         PCI Express Root Port
/0/100/18.4                    bridge         PCI Express Root Port
/0/100/18.5                    bridge         PCI Express Root Port
/0/100/18.6                    bridge         PCI Express Root Port
/0/100/18.7                    bridge         PCI Express Root Port
/1                 scsi32      storage        
/2                 input0      input          Power Button
/3                 input1      input          AT Translated Set 2 keyboard
/4                 input3      input          VirtualPS/2 VMware VMMouse
/5                 input4      input          VirtualPS/2 VMware VMMouse
/6                 input5      input          VMware VMware Virtual USB Mouse
/7                 input6      input          VMware DnD UInput pointer
```

``` shell
dpkg -l | grep libva
>> no output
```

``` shell
./aspect-model-editor
>> no output, same behavior as described above > software freezes

type control +c
^CError sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94617)
    at WebContents.send (node:electron/js2c/browser_init:2:79767)
    at #handleClosingWindow (/home/tommeyer/AAS/aspect-model-editor-v5.1.1-linux/resources/app.asar/electron-libs/windows-manager.js:217:24)
    at BrowserWindow.<anonymous> (/home/tommeyer/AAS/aspect-model-editor-v5.1.1-linux/resources/app.asar/electron-libs/windows-manager.js:174:32)
    at BrowserWindow.emit (node:events:513:28)

```

**Comment by michelu89** [comment]
Hi @tom-rm-meyer-ISST 

We have just released a new version, [5.2.0](https://github.com/eclipse-esmf/esmf-aspect-model-editor/releases/tag/v5.2.0), which includes several adjustments. We've also updated Electron to incorporate various fixes related to the Linux environment, as detailed here: [28.2.1](https://github.com/electron/electron/releases/tag/v28.2.1). Could you please test and see if this resolves your issue?

**Comment by tom-rm-meyer-ISST** [comment]
@michelu89 thanks a lot! version 5.2.0 fixed the issue and now seems to work! :)
